softdev.bib

```diff
@@ -2757,3 +2757,43 @@ @misc{watson21assessing
     month = Sep,
     url = "https://www.capabilitieslimited.co.uk/pdfs/20210917-capltd-cheri-desktop-report-version1-FINAL.pdf"
 }
+
+@misc{servo,
+    title = {The Servo Parallel Browser Engine Project},
+    howpublished = {\href{https://servo.org}{https://servo.org}},
+    note = {Accessed: 2022-12-17},
+    year = {2012},
+    author = {Mozilla},
+}
+
+@misc{rustgc,
+    title = {rust-gc: simple tracing (mark and sweep) garbage collector for Rust},
+    howpublished = {\href{https://github.com/manishearth/rust-gc/}{https://github.com/manishearth/rust-gc/}},
+    note = {Accessed: 2022-12-17},
+    year = {2015},
+    author = {Manish Goregaokar},
+}
+
+@misc{shifgrethor,
+    title = {shifgrethor},
+    howpublished = {\href{https://github.com/withoutboats/shifgrethor/}{https://github.com/withoutboats/shifgrethor/}},
+    note = {Accessed: 2022-12-17},
+    year = {2018},
+    author = {withoutboats},
+}
+
+@misc{luster,
+    title = {luster - An experimental Lua VM implemented in pure Rust},
```

Comment:
I can't remember if the outer curlies `{...}` solve this, but you might want to check whether for proper nouns (e.g. "Luster", "Lua") and abbrevations (VM) we need inner curlies around them or not.

Reply:
I've fixed them here: e039b2b5ac3eb203f1cbb1454874862ccb47fc1d